src/other/pages/guides/getting-started/story.jsx

```diff
@@ -0,0 +1,228 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactSyntaxHighlighter from 'react-syntax-highlighter';
+import style from 'react-syntax-highlighter/dist/esm/styles/hljs/github-gist';
+import { AXAText, AXAHeading } from './plib-components';
+import styles from './index.scss';
+
+export default {
+  title: 'Guides/Getting started',
+  decorators: [],
+  parameters: {
+    knobs: { disabled: true },
+    changelog: { disabled: true },
+    codepreview: { disabled: true },
```

Comment:
Changelog and codepreview do no longer exist

Reply:
No, they are just deactivated. Hope to get them running soon.

Reply:
This is on purpose, they are not needed and the pages look better without. I'm trying to not show the addons bar completely.

Reply:
hide panels done

Reply:
It seems to not work correctly? On initial start of SB theres still "nothing found". But if i open up another story and go back to "welcome" story, the panel is gone?!